Cargo.toml

```
@@ -7,30 +7,31 @@ edition = "2021"
 
 [dependencies]
 atomic = "0.6.0"
-base64 = "0.21.4"
-bcrypt = "0.15.0"
-bigdecimal = "0.4.1"
-bitflags = { version = "2.4.0", features = ["serde"] }
-chrono = { version = "0.4.31", features = ["serde"] }
+base64 = "0.22.1"
+bcrypt = "0.15.1"
+bigdecimal = "0.4.3"
+bitflags = { version = "2.5.0", features = ["serde"] }
+chrono = { version = "0.4.38", features = ["serde"] }
 dotenv = "0.15.0"
-futures = "0.3.28"
-hostname = "0.3.1"
-jsonwebtoken = "8.3.0"
+futures = "0.3.30"
+hostname = "0.4.0"
+jsonwebtoken = "9.3.0"
 lazy_static = "1.4.0"
-log = "0.4.20"
-log4rs = { version = "1.2.0", features=["rolling_file_appender", "compound_policy", "size_trigger", "gzip"] }
-num-bigint = "0.4.4"
-num-traits = "0.2.16"
-openssl = "0.10.57"
-poem = "1.3.58"
+log = "0.4.21"
+log4rs = { version = "1.3.0", features = ["rolling_file_appender", "compound_policy", "size_trigger", "gzip"] }
+num-bigint = "0.4.5"
+num-traits = "0.2.19"
+openssl = "0.10.64"
+poem = "3.0.1"
 rand = "0.8.5"
-regex = "1.9.5"
-reqwest = "0.11.20"
-serde = { version = "1.0.188", features = ["derive"] }
-serde_json = { version = "1.0.107", features = ["raw_value"] }
-sqlx = { version = "0.7.1", features = ["json", "chrono", "ipnetwork", "runtime-tokio-native-tls", "any"] }
-thiserror = "1.0.48"
-tokio = { version = "1.32.0", features = ["full"] }
-chorus = { git="https://github.com/Zert3x/chorus", branch = "misc/backend-stuff", features = ["backend"], default-features = false } #git="https://github.com/Zert3x/chorus" path="../chorus"
-sentry = "0.31.7"
-clap = { version = "4.4.6", features = ["derive"] }
+regex = "1.10.4"
+reqwest = "0.12.4"
+serde = { version = "1.0.203", features = ["derive"] }
+serde_json = { version = "1.0.117", features = ["raw_value"] }
+sqlx = { version = "0.7.4", features = ["json", "chrono", "ipnetwork", "runtime-tokio-native-tls", "any"] }
+thiserror = "1.0.61"
+tokio = { version = "1.38.0", features = ["full"] }
+sentry = "0.33.0"
+clap = { version = "4.5.4", features = ["derive"] }
+
+chorus = { git = "ssh://git@github.com/polyphony-chat/chorus", branch = "dev", features = ["backend"], default-features = false } # git = "ssh://git@github.com/Quat3rnion/chorus" # path = "../chorus"
```

Try using https to fix the tests:
```suggestion
chorus = { git = "https://github.com/polyphony-chat/chorus", branch = "dev", features = ["backend"], default-features = false } # git = "https://github.com/Quat3rnion/chorus" # path = "../chorus"
```

---

Ah true, I'll change that in my next commit